apps/frontend/src/pages/projects/components/project-create-form/project-create-form.tsx

```diff
@@ -13,12 +19,20 @@ type Properties = {
 };
 
 const ProjectCreateForm = ({ onSubmit }: Properties): JSX.Element => {
-	const { control, errors, handleSubmit } = useAppForm<ProjectCreateRequestDto>(
-		{
+	const { control, errors, handleSubmit, trigger } =
+		useAppForm<ProjectCreateRequestDto>({
 			defaultValues: DEFAULT_PROJECT_CREATE_PAYLOAD,
 			validationSchema: projectCreateValidationSchema,
-		},
-	);
+		});
+
+	const descriptionValue = useFormWatch({
+		control,
+		defaultValue: "",
+		name: "description",
+	});
+
+	const isDescriptionCounterShown =
+		descriptionValue.length <= ProjectValidationRule.DESCRIPTION_MAXIMUM_LENGTH;
```

Comment:
Maybe it would be better to check it this way:
```suggestion
	const isDescriptionCounterShown = Boolean(errors[description]?.message)
```